### PR TITLE
Combat harmonization

### PR DESCRIPTION
**Pull request by baptisp**
Merge request pour la review de jeudi 28/11/2024 sur la méthode d'harmonisation ComBat.

Le papier original est assez long, il y a beaucoup d'expériences et de résultats pour montrer que ComBat est plutôt fiable par rapport à l'état de l'art. Peut-être que tout n'est pas nécessaire.

Tout critique ou suggestion d'amélioration est la bienvenue pour ce premier post du doctorant !

### CODE REVIEW

File: collections/_posts/2024-11-28-ComBat-Harmonization.md [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+The RAVEL algorithm attempts to estimate a voxel-specific unwanted variation term by using a control region of the brain to estimate latent factors of unwanted variation common to all voxels.
+
+The SVA algorithm estimates latent factors of unwanted variation, called surrogate variables, that are not associated with the biological covariates of interest. It is particularly useful when the site variable is not known, or when there exists residual unwanted variation after the removal of site effects
+
+
+### ComBat
+ComBat model was introduced in the context of gene expression analysis by Johnson et al. [2007] [^6] as an improvement of location/scale models for studies with small sample size.
+The ComBat model is reformulated in the context of DTI images.
+Let $m$ the number of imaging sites, containing $n_i$ for $i=1,2,...,m$ volume scans.
+For voxel $\nu=1,2,...,p$, let $y_{ij\nu}$ represent the diffusion metric measure (FA for e.g.) at voxel $\nu$ for the scan $j$ at site $i$. ComBat posits the fllowing location and scale (L/S) adjustement model:
```
Comment (bayaca) [inline collections/_posts/2024-11-28-ComBat-Harmonization.md:101]:
```suggestion
For voxel $\nu=1,2,...,p$, let $y_{ij\nu}$ represent the diffusion metric measure (FA for e.g.) at voxel $\nu$ for the scan $j$ at site $i$. ComBat posits the following location and scale (L/S) adjustement model:
```

File: collections/_posts/2024-11-28-ComBat-Harmonization.md [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+
+The final ComBat-harmonized diffusion metric values are defined as:
+
+$$ y_{ijv}^{ComBat} = \frac{y_{ij\nu}-\hat\alpha_\nu -X_{ij}\hat\beta_\nu - \gamma_{i\nu}^*}{\delta_{i\nu}^*} + \hat\alpha_\nu + X_{ij}\hat\beta_\nu \qquad (4)$$
+
+
+### How does it work in practice ?
+Another bibliography session ?
+
+<!-- Show how the compute the estimates with the bayesian framework -->
```
Comment (bayaca) [inline collections/_posts/2024-11-28-ComBat-Harmonization.md:128]:
```suggestion
<!-- Show how to compute the estimates with the bayesian framework -->
```

File: collections/_posts/2024-11-28-ComBat-Harmonization.md [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+
+To further evaluate the performance of the different harmonization methods, two sets of silver-standards per site were created:
+1) A silver-standard for voxels that are truly associated with age (signal silver-standard).
+2) A silver-standard for voxels that are not associated with age (null silver-standard).
+
+For each silvert-standard two different sets are created since previous studies have shown that some brain regions are more specific in changes for FA and others for MD.
+
+To estimate them, the authors use a meta-analytic approach: 
+1) for each site separatly, at each voxel in the WM, they apply a linear regression model to obtain a t-statistic measuring the association of FA with age.
+2) For each site, they define the site-specific signal silver-standard to be the k=5000 voxels with the highest t-sttistics in magnitude.
```
Comment (bayaca) [inline collections/_posts/2024-11-28-ComBat-Harmonization.md:157]:
```suggestion
2) For each site, they define the site-specific signal silver-standard to be the k=5000 voxels with the highest t-statistics in magnitude.
```

File: collections/_posts/2024-11-28-ComBat-Harmonization.md [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+* RAVEL, SVA and ComBat reduce greatly the inter-sites differences.
+* Ravel does not seem to account for local-site effects.
+
+<!-- understand why Ravel downperforms -->
+
+<!--  -->
+<div style="text-align:center">
+<img src="/collections/images/comBat/fig4.jpg" width=800></div>
+
+Figure 4a shows a t-statistic at each voxel to measure the association of the DTI scalar values with site. A voxel is significant is the p-value calculated from the two-sample t-test is less than 0.05, after correcting for multiple comparisons using Bonferroni correction. They authors don't explain how they realize the correction in practice. For more details about how they perform a two-sample t-statistics with DTI data, see Figure A.3.
```
Comment (bayaca) [inline collections/_posts/2024-11-28-ComBat-Harmonization.md:188]:
```suggestion
Figure 4a shows a t-statistic at each voxel to measure the association of the DTI scalar values with site. A voxel is significant if the p-value calculated from the two-sample t-test is less than 0.05, after correcting for multiple comparisons using Bonferroni correction. They authors don't explain how they realize the correction in practice. For more details about how they perform a two-sample t-statistics with DTI data, see Figure A.3.
```

File: collections/_posts/2024-11-28-ComBat-Harmonization.md [hunk 5]
```diff
@@ -0,0 +1,358 @@
...
+This part aims to investigate the robustness of the different harmonization techniques to datasets for which age is confounded by site.
+
+* The two main datasets are carefully **matched** for age, gender, ethnicity to minimize potential confounding of those kind of variables with site. Howether matching has limitations as it strongly depends on the overlap of covariates between datasets. Poor overlap means many scans to be excluded, making matching infeasible in many applications where harmonization is needed.
+
+* Howether doing nothing will let an undesirable situation happened where site will be a confounder for the relationship between the DTI values and the phenotypes of interest.
+  
+* Confounding between age and site presents an additional challenge for harmonization, since removing variation associated site can lead to removing variation associated with age if not done carefully.
+
+To evaluate the robustness of the different harmonization methods in the presences of statistical  confounding between imaging site and age (that is when age is unbalanced with respect to site), the authors selected different subsets of the data to create confounding scenarios.
```
Comment (bayaca) [inline collections/_posts/2024-11-28-ComBat-Harmonization.md:234]:
```suggestion
To evaluate the robustness of the different harmonization methods in the presences of statistical confounding between imaging site and age (that is when age is unbalanced with respect to site), the authors selected different subsets of the data to create confounding scenarios.
```

File: collections/_posts/2024-11-28-ComBat-Harmonization.md [hunk 6]
```diff
@@ -0,0 +1,358 @@
...
+</div>
+
+<div style="text-align:center">
+<img src="/collections/images/comBat/fig7.jpg" width=800></div>
+
+<div style="text-align:center">
+Figure 7 presents the distribution of the estimated effect sizes on the signal silver-standard for all methods and for all confounding scenarios. Th dashed line represents the median of the true effect sizes, and the solid line represents an effect size of 0.
+</div>
+
+Results for raw data are consitent: positive (resp. negative) confoundind shifts the effect size positively (resp negatively).
```
Comment (bayaca) [inline collections/_posts/2024-11-28-ComBat-Harmonization.md:295]:
```suggestion
Results for raw data are consitent: positive (resp. negative) confounding shifts the effect size positively (resp negatively).
```

File: collections/_posts/2024-11-28-ComBat-Harmonization.md [hunk 7]
```diff
@@ -0,0 +1,358 @@
...
+
+Figure 9.c shows the densities of the t-statistics for the voxels not associated with age (null voxels) for the FA maps.
+* A good harmonization method should result in t-statistics centered around 0.
+* Performance is variable across harmonization methods. But ComBat is one of those who do perform correctly.
+
+The authors investigated the stability of the ComBat harmonization parameters by running ComBat on random subsamples of variable size. They obtained that site effects estimated from subsamples approximate well the site effects estimated from the full dataset.
+
+# Discussion
+
+* The authors shows that combining the two studies without proper harmonization led to a decrease in power od detecting voxels associated with age.
```
Comment (bayaca) [inline collections/_posts/2024-11-28-ComBat-Harmonization.md:337]:
```suggestion
* The authors show that combining the two studies without proper harmonization led to a decrease in power of detecting voxels associated with age.
```

**Review by bayaca — CHANGES_REQUESTED** [review]
very in-depth an thorough review, well done !

for the oral presentation, maybe focus on the intuition behind the technical details, as you did in the discussion section, and invite the listeners to refer to the post for further details

File: collections/_posts/2024-11-28-ComBat-Harmonization.md [hunk 8]
```diff
@@ -0,0 +1,358 @@
...
+     title:   "Harmonization of multi-site diffusion tensor imaging data"
+     venue:   "article from NeuroImage"
+ pdf: "https://www.biorxiv.org/content/10.1101/116541v3.full.pdf"
+ ---
+
+# Highlights
+
+* Significant site and scanner effects exist in DTI scalar maps.
+* Several multi-site harmonization methods are proposed.
+* ComBat performs the best at removing site effects in FA and MD.
```
[the remark this answers is not in the document]
Comment (bayaca) [reply collections/_posts/2024-11-28-ComBat-Harmonization.md:16]: same comment for the other metrics

File: collections/_posts/2024-11-28-ComBat-Harmonization.md (hunk 7 above)
Comment (MathildeDupouy) [inline collections/_posts/2024-11-28-ComBat-Harmonization.md:337]: power **of** detecting

**Review by MathildeDupouy — APPROVED** [review]
Thanks for this really extensive review !

on my opinion, the points to outline for the oral presentations will be the technique in itself and also the different ways of evaluating an harmonization methods. I think you can even add an highlight like "An extensive evaluation and comparison of several methods is carefully conducted" because I feel like this is a strength of the paper, isn't it ?

I hope my reviews will be helpful and I am looking forward for your explanations on Thursday !

**Review by bayaca — APPROVED** [review]
👍

**Pull request merged**